README.md

```diff
@@ -414,6 +415,17 @@ to keep in mind when using Nimble in Objective-C:
 
    expect(@(1 + 1)).to(equal(@2));
    expect(@"Hello world").to(contain(@"world"));
+
+   // Boxed as NSNumber*
+   expect(2).to(equal(2));
+   expect(1.2).to(beLessThan(2.0));
+   expect(true).to(beTruthy());
+
+   // Boxed as NSString *
```

Comment:
Tiny nit-pick: Above you use `NSNumber*` (no space), here you use `NSString *` (space between type and pointer).